eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -54,6 +54,39 @@
       "RunAggregateReports": false,
       "TestGoals": "surefire:test",
       "TestOptions": ""
+    },
+    {
+      "Agent": {
+        "ubuntu-20.04": { "OSVmImage": "MMSUbuntu20.04", "Pool": "azsdk-pool-mms-ubuntu-2004-general" }
+      },
+      "JavaTestVersion": "1.17",
+      "AZURE_TEST_HTTP_CLIENTS": "jdk-httpclient",
+      "TestFromSource": true,
+      "RunAggregateReports": false,
+      "TestGoals": "verify",
+      "TestOptions": "-DskipCompile=true -DskipTestCompile=true -DcreateSourcesJar=false"
+    },
+    {
+      "Agent": {
+        "windows-2022": { "OSVmImage": "windows-2022", "Pool": "azsdk-pool-mms-win-2022-general" }
+      },
+      "JavaTestVersion": "1.17",
+      "AZURE_TEST_HTTP_CLIENTS": "jdk-httpclient",
+      "TestFromSource": false,
+      "RunAggregateReports": true,
+      "TestGoals": "verify",
+      "TestOptions": "-DskipCompile=true -DskipTestCompile=true -DcreateSourcesJar=false"
+    },
+    {
+      "Agent": {
+        "ubuntu-20.04": { "OSVmImage": "MMSUbuntu20.04", "Pool": "azsdk-pool-mms-ubuntu-2004-general" }
+      },
+      "JavaTestVersion": "1.17",
+      "AZURE_TEST_HTTP_CLIENTS": "jdk-httpclient",
+      "TestFromSource": false,
+      "RunAggregateReports": false,
+      "TestGoals": "surefire:test",
+      "TestOptions": ""
```

Comment:
Do we need three of these runs? Could this be cut down to two additional runs of Linux and Windows? Right now, this change is adding 4 additional runs to live testing which is a lot more testing.

Reply:
Yeah. If it is time-consuming we can think to have testing Linux only if applies.

Reply:
There's also a cost/pipeline agent resourcing factor besides time.